src/openrct2/windows/map.c

```diff
@@ -176,7 +176,7 @@ static uint8 _activeTool;
 static uint32 _currentLine;
 
 /** rct2: 0x00F1AD68 */
-static uint8 (*_mapImageData)[512][512];
```

Comment:
I remember it crashed when this was not updated (and I changed the defined value)